orchestrator-salt/src/main/resources/salt/salt/sssd/ipa.sls

```diff
@@ -65,7 +65,7 @@ removing_dns_entries:
 
 add_dns_record:
   cmd.run:
-    - name: /opt/salt/scripts/add_dns_record.sh && echo $(date +%Y-%m-%d:%H:%M:%S) >> /var/log/dnsrecord-add-executed
+    - name: /opt/salt/scripts/add_dns_record.sh 2>&1 | tee -a /var/log/dnsrecord-add-history ; test ${PIPESTATUS[0]} -eq 0 && echo $(date +%Y-%m-%d:%H:%M:%S) >> /var/log/dnsrecord-add-executed
```

Comment:
- is there a reason you used `;` instead of `&&`?
- I think `dnsrecord-add.log` would be better name, and more consistent with the other salt states, as we use the `.log` for logs and the `executed` ending for indicating that the script has run

Reply:
`;` and && are same here. I changed it to && now. I feel ; would make the one line only separated by a single &&. Therefore, it is clear that first block "/opt/salt/scripts/add_dns_record.sh 2>&1 | tee -a /var/log/dnsrecord-add-history ; test ${PIPESTATUS[0]} -eq 0 " was a whole section. But this is more of a style preference. 

Thanks! I changed the log name to "dnsrecord-add.log"